clients/drcachesim/tests/raw2trace_unit_tests.cpp

```diff
@@ -105,8 +80,8 @@ class raw2trace_test_t : public raw2trace_t {
                       // debugging and viewing of what's going on.
                       4)
     {
-        module_mapper_ =
-            std::unique_ptr<module_mapper_t>(new module_mapper_test_t(instrs, drcontext));
+        module_mapper_ = std::unique_ptr<module_mapper_t>(
+            new test_module_mapper_t(&instrs, drcontext));
```

Comment:
(It's weird that this re-formatted when the name is the same length.)

Reply:
`instrs` was changed to `&instrs` so the line length increased.